### PR TITLE
SCC-757 - added pre market event flag for event type

### PR DESCRIPTION
**Pull request by sudhakar-reddy-konda**

### CODE REVIEW

File: src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/model/dto/EventType.java [hunk 1]
```diff
@@ -12,4 +12,6 @@
 
   private String description;
 
+  private  Boolean isPreMarketEvent;
```
Comment (tommyb82) [inline src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/model/dto/EventType.java:15]: I think the property name should be `preMarketActivity` - see https://github.com/Crown-Commercial-Service/ccs-scale-api-definitions/pull/81/files#diff-046ce7653c0a5c01235e166008bc0cc5edcc7f00152be4dadc4cd264b1f81ba7R791

File: src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/model/entity/ProcurementEventType.java [hunk 2]
```diff
@@ -25,4 +25,11 @@
 
   @Column(name = "procurement_event_type_name")
   String name;
+
+  @Column(name = "procurement_event_type_description")
+  String description;
+
+  @Column(name = "premarket_activity_ind")
+  Boolean isPreMarketEvent;
```
Comment (tommyb82) [inline src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/model/entity/ProcurementEventType.java:33]: As above - `isPremarketActivity` would seem a better match 👍

File: src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/model/dto/EventType.java (hunk 1 above)
Comment (sudhakar-reddy-konda) [reply src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/model/dto/EventType.java:15]: Done

File: src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/model/entity/ProcurementEventType.java (hunk 2 above)
Comment (sudhakar-reddy-konda) [reply src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/model/entity/ProcurementEventType.java]: Done

**Review by tommyb82 — APPROVED** [review]
Looks good - can see updated structure and content in response 👍

**Pull request merged**